ur_moveit_config/launch/ur_moveit.launch.py

```diff
@@ -162,6 +163,11 @@ def launch_setup(context, *args, **kwargs):
 
     # Trajectory Execution Configuration
     controllers_yaml = load_yaml("ur_moveit_config", "config/controllers.yaml")
+    # the scaled_joint_trajectory_controller does not work on simulated robots
```

Comment:
```suggestion
    # the scaled_joint_trajectory_controller does not work on fake hardware
```

This does not necessarily affect other simulations such as URsim or gazebo.